pom.xml

```diff
@@ -98,10 +98,10 @@
         <gelfclient.version>1.4.1</gelfclient.version>
         <grok.version>0.1.7-graylog</grok.version>
         <guava-retrying.version>2.0.0</guava-retrying.version>
-        <guava.version>23.4-jre</guava.version>
+        <guava.version>23.5-jre</guava.version>
         <guice.version>4.1.0</guice.version>
         <HdrHistogram.version>2.1.10</HdrHistogram.version>
-        <hibernate-validator.version>6.0.4.Final</hibernate-validator.version>
+        <hibernate-validator.version>6.0.5.Final</hibernate-validator.version>
```

Comment:
I am seeing the following log message:

```
2018-01-04 16:38:33,805 INFO : org.hibernate.validator.internal.util.Version - HV000001: Hibernate Validator 5.1.3.Final
```

Is there an issue with transitive dependencies?

Reply:
This was a result of the artifact relocation into another group:
https://developer.jboss.org/wiki/HibernateValidatorMigrationGuide?_sscc=t#jive_content_id_600Final